packages/docs/README.md

```diff
@@ -1,19 +1,69 @@
-# docs
+<a name="readme-top"></a>
 
-This is a Next.js application generated with
-[Create Fumadocs](https://github.com/fuma-nama/fumadocs).
+<div align="center">
+  <img src="../../assets/PNGs/icon.png" alt="Logo" width="180">
+  <h1 align="center"> Unified Backend for APIs, Events, and Agents </h1>
+</div>
 
-Run development server:
+<div align="center">
+  <a href="https://motia.dev"><img src="https://img.shields.io/badge/PROJECT-PAGE-FFE165?style=for-the-badge&labelColor=555555" alt="Project Page"></a>
+  <a href="https://discord.gg/nJFfsH5d6v"><img src="https://img.shields.io/badge/DISCORD-JOIN%20US-9146FF?style=for-the-badge&labelColor=555555" alt="Discord"></a>
+  <a href="https://motia.dev/docs"><img src="https://img.shields.io/badge/DOCS-READ%20NOW-000000?style=for-the-badge&labelColor=555555" alt="Documentation"></a>
+  <a href="https://www.npmjs.com/package/motia"><img src="https://img.shields.io/npm/v/motia?style=for-the-badge&label=NPM&labelColor=555555&color=CB3837" alt="NPM Version"></a>
+  <a href="https://www.npmjs.com/package/motia"><img src="https://img.shields.io/npm/dt/motia?style=for-the-badge&label=DOWNLOADS&labelColor=555555&color=CB3837" alt="NPM Downloads"></a>
+  <a href="#"><img src="https://img.shields.io/badge/STATUS-BETA-FFE165?style=for-the-badge&labelColor=555555" alt="Status Beta"></a>
+  <hr>
+</div>
 
-```bash
-npm run dev
-# or
-pnpm dev
-# or
-yarn dev
-```
+---
 
-Open http://localhost:3000 with your browser to see the result.
+Motia is a modern backend framework that combines APIs, background jobs, event systems, and AI agents into one unified, observable runtime. Stop context-switching between multiple runtimes—build your entire backend with event-driven steps, mixing JavaScript, TypeScript, and Python freely, while keeping shared state, tracing, and deployment simplicity.
+
+---
+
+## 📝 Motia Documentation
+
+Welcome to the source code of [the official Motia docs](http://motia.dev/docs/).
+Motia documentation has been created with [Create Fumadocs](https://github.com/fuma-nama/fumadocs). Join [the Motia community](https://discord.gg/m3XN23Kp) and follow along to see how you can contribute.
+
+1. Join [the Motia community](https://discord.gg/m3XN23Kp) to ask questions, interact with the core Motia team and your fellow community members! ❤️
+2. Fork the Motia repository and clone your fork to your local machine. 🖥️ 
+3. Open the root directory and navigate to the docs directory using this command:
+
+    ``` bash
+      cd packages/docs 
+    ```
+
+4. Install all the packages:
+
+    ``` bash
+      pnpm i
+      # or
+      npm i
+    ```
```

Comment:
it should be only `pnpm install` in the root of the project, which should take care of everything